dashboard/.gitignore

```diff
@@ -1,3 +1,5 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
 
```

Comment:
Looks like you didn't get all of your merge conflicts resolved. The `<<<<<<< HEAD`, `======` and `>>>>>>>>` lines all signal conflicts.

Reply:
All merge conflicts related to gitignore file are resolved.